client/extensions/woocommerce/components/product-image-uploader/index.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import { head, uniqueId, find, noop } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { errorNotice as errorNoticeAction } from 'state/notices/actions';
+
+import DropZone from 'components/drop-zone';
+import FilePicker from 'components/file-picker';
+import MediaActions from 'lib/media/actions';
+import MediaUtils from 'lib/media/utils';
+import MediaStore from 'lib/media/store';
+import MediaValidationStore from 'lib/media/validation-store';
+
+class ProductImageUploader extends Component {
+	static propTypes = {
+		siteId: PropTypes.number,
+		multiple: PropTypes.bool,
+		compact: PropTypes.bool,
+		onSelect: PropTypes.func,
+		onUpload: PropTypes.func,
+		onError: PropTypes.func,
+		onFinish: PropTypes.func,
+	};
+
+	static defaultProps = {
+		multiple: true,
+		compact: false,
+		onSelect: noop,
+		onUpload: noop,
+		onError: noop,
+		onFinish: noop,
+	}
+
+	state = {
+		errors: [],
+	};
+
+	showError = ( media, transientId ) => {
+		const { onError, errorNotice, translate } = this.props;
+		const { errors } = this.state;
+
+		onError( {
+			file: media,
+			transientId,
+		} );
+
+		let extraDetails;
+		const validationError = errors[ transientId ] || [];
+		switch ( head( validationError ) ) {
+			case 'EXCEEDS_MAX_UPLOAD_SIZE' :
+				extraDetails = translate( 'This file exceeds the maximum upload size.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'This file exceeds the maximum upload size for this site.' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woocommerce-product-images)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).